# Super-RF: a new efficient metric to compare phylogenetic trees defined on different but partially overlapping set of leaves
## Authors: Olivier Desmarais and Nadia Tahiri


## Abstract:

__Background.__ Understanding the successive changes in the genome (i.e., horizontal gene transfer or hybridization) of living organisms is very useful in many fields such as comparative and evolutionary biology. This is the purpose of phylogeny, which expresses these evolutionary events chronologically in the form of a phylogenetic tree, where each leaf of the tree corresponds to a species, introduced by the famous paper of Robinson and Fould in 1981. Studies propose to merge several phylogenetic trees, each representing the evolutionary history of a specific gene, to obtain a better representation of these evolutionary events. This fusion can result in one or more phylogenetic trees to maximize the resolution of the resulting trees.
Current knowledge allows the comparison of multiple phylogenetic trees with sets of phylogenetic trees defined on the same sets of species, thus allowing the construction of a consensus tree or alternative consensus trees. In this process, the differences between each tree are calculated using a distance metric, such as the Robinson and Foulds distance (RF distance) or the least squares distance (LS distance). The RF distance, which is of interest to us in this article, consists of examining the bipartitions of the profile of the two trees being compared, i.e., the exhaustive list of subtrees they contain. The distance between the two trees is then established according to the differences of these profiles. The distance obtained can then be normalized using an equation based on the number of common leaves between the two trees.
To date, the literature does not provide a robust and efficient method for constructing a supertree, which is equivalent to a consensus tree, but in the context where it is constructed from sets of phylogenetic trees defined on different but overlapping species. This is mainly due to the fact that there is no metric to compare phylogenetic trees in this specific context. However, most of the time, the genomic datasets available in the databases are not defined on the same sets of species. Therefore, the development of a new metric capable of comparing phylogenetic trees defined on different but overlapping species is necessary.


__Methodology.__ To address this problematic, we developed a metric able to establish the distance between two phylogenetic trees, T1 and T2, when the datasets of those trees are defined on different but overlapping species. For this purpose, we adapted the normalized RF distance in a way that handle the distance between two trees in the studied context. Those adaptations rely mainly on two aspects: (1) the attribution of an ÔÅ° penalty for every partition unique to the profile of a specific tree and (2) the consideration of both the common and non-common leaves in the normalization of the metric. We called this new metric the super-RF, since it can be applied in the context of super-trees.
We tested our new super-RF metric on genomic data from methanogens, organisms capable of producing methane from simple carbon compounds (CO2, CO, acetate, formate, methyl groups, etc.) and hydrogen. They are classified within the archaea into two classes and six major orders. The choice of these data was primarily motivated by the uncertainties surrounding the relationships between orders within each class. Therefore, we prepared the data following a rigorous protocol, including sequence collection, sequence alignment, identification of unique genes and finally inference of phylogenetic trees.


__Results.__ The average of supertree filtered to retain only majority edges produces a tree with a resolution that is lower than the phylogeny inferred by a supermatrix approach, but higher than that obtained by approaches using supertree majority. Furthermore, the resulting filtered average supertree is consistent with the reference tree obtained by supermatrix. No false edges are proposed. Moreover, the topology of the filtered supertree is very close to the reference phylogeny, even in the case of incomplete gene families. We tested the case of gene families where on average 10% of the sequences had been deleted before the analysis. Further analyses are underway to determine the behavior of this type of supertree as the number of missing data increases.
We have proved by mathematical foundations the stability of our new super-RF metric applied in the case of alternative supertrees. These trees have the advantage of being better solved than traditional supertrees (i.e., unique supertree), since they are able to solve its multi-furcations. Therefore, more information can be extracted from the input trees. 


__Conclusion.__ We have shown that it is possible to calculate the Robinson-Foulds distance between phylogenetic trees with different but partially overlapping species sets using our new metric called the super-RF distance. This metric is relevant for comparing trees that do not have the same species set, which is a major problem in the global Tree of Life project. This metric could therefore be incorporated into the construction of alternative supertrees, more resolved than those obtained by classical approaches.
From there, it could be interesting to explore the possibility of adapting other metrics to allow them to play the same role as the super-RF distance. For example, the Least-Squares distance would be an excellent candidate since it takes into account the length of the branches of phylogenetic trees.
